+bert/+tokenizer/+internal/FullTokenizer.m

```diff
@@ -85,12 +106,16 @@
             %   tokens = tokenize(tokenizer,text) tokenizes the input
             %   string text using the FullTokenizer specified by tokenizer.
             basicToks = this.Basic.tokenize(txt);
-            basicToksUnicode = textanalytics.unicode.UTF32(basicToks);
-            subToks = cell(numel(basicToks),1);
-            for i = 1:numel(basicToks)
-                subToks{i} = this.WordPiece.tokenize(basicToksUnicode(i));
+            basicToksUnicode = cellfun(@textanalytics.unicode.UTF32,basicToks,UniformOutput=false);
+            toks = cell(numel(txt),1);
+            for i = 1:numel(txt)
```

Comment:
At a glance - can this work with one loop, i.e. do the `UTF32` conversion in the `for` loop? Would save looping over `txt`/`basicToks` twice.